src/game/logic/system/partitionmanager.h

```diff
@@ -119,10 +120,28 @@ class SightingInfo : public MemoryPoolObject, public SnapShot
 {
     IMPLEMENT_POOL(SightingInfo);
 
+    void *operator new(size_t size, void *dst) { return dst; }
```

Comment:
Remove these, these are not needed with the #include <new> in there.

Reply:
Seems to be needed still. Getting a compile error otherwise. Tbh i don't think we need to hook the ctor I might just remove that.